hooks/pre-commit

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+RED="\033[1;31m"
+GREEN="\033[1;32m"
+NC="\033[0m"
+linter_exit_code=1
+cd client && npm run lint
```

Comment:
Can we also run `npm run pretty` for both client and server?

Reply:
Added.